tests/PHPUnit/Fixtures/SqlDump.php

```diff
@@ -68,6 +69,12 @@ public function setUp(): void
         $defaultsFile = $this->makeMysqlDefaultsFile($user, $password);
 
         $cmd = "mysql --defaults-extra-file=\"$defaultsFile\" -h \"$host\" {$this->dbName} < \"" . $deflatedDumpPath . "\" 2>&1";
+
+        if (DatabaseConfig::getConfigValue('schema') === 'Tidb') {
+            // For TiDb we need to remove the default charset from the create table statements, otherwise it will use the default charset collation, which differs from database default collation
+            $cmd = "sed 's/ DEFAULT CHARSET=utf8mb4//g' \"" . $deflatedDumpPath . "\" | mysql --defaults-extra-file=\"$defaultsFile\" -h \"$host\" {$this->dbName}  2>&1";
```

Comment:
```suggestion
            $cmd = "sed 's/ DEFAULT CHARSET=utf8mb4//' \"$deflatedDumpPath\" | mysql --defaults-extra-file=\"$defaultsFile\" -h \"$host\" {$this->dbName} 2>&1";
```

Mostly for the `/g` modifier in the `sed` command, that one should not be necessary unless we expect there to be multiple `CREATE TABLE` statements in a single line.

And we shouldn't mix more interpolation styles in one line :see_no_evil: